content/pt-br/docs/concepts/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Proxies no Kubernetes
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+Esta página descreve o uso de proxies com Kubernetes. 
+
+
+<!-- body -->
+
+## Proxies
+Existem vários tipos diferentes de proxies que você pode encontrar usando Kubernetes:
+
+
+1.  O [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - executa na máquina do usuário ou em um pod
+    - proxies de um host local endereçam para o servidor de API do kubernetes
+    - cliente para o proxy usa HTTP 
+    - proxy para o servidor de API usa HTTPS 
+    - localiza o servidor de API
+    - adiciona os cabeçalhos de comunicação.
+
+1.  O [apiserver proxy](/docs/tasks/access-application-cluster/access-cluster/#discovering-builtin-services):
+
+    - é um bastion server, construído no servidor de API
+    - conecta um usuário fora do cluster com os IPs do cluster que não podem ser acessados de outra forma
+    - executa dentro do processo do servidor de API
+    - cliente para proxy usa HTTPS (ou http se o servidor de API for configurado)
+    - proxy para o destino pode usar HTTP ou HTTPS conforme escolhido pelo proxy usando as informações disponíveis
+    - pode ser usado para acessar um Nó, Pod ou serviço
+    - faz balanceamento de carga quando usado para acessar um Service.
+
+1.  O [kube proxy](/docs/concepts/services-networking/service/#ips-and-vips):
+
+    - executa em todos os Nós 
+    - atua como proxi para UDP, TCP e SCTP
+    - não aceita HTTP
+    - provém balanceamento de carga
+    - apenas é usado para acessar serviços.
+
+1.  Um Proxy/Balanceador de carga na frente de servidores de API(s):
+
+    - existência e implementação variam de cluster para cluster, por exemplo  nginx
```

Comment:
This phrase is strange. What we are trying to tell here?

Reply:
I have tried to add more clarity here: `a existência e a implementação de tal elemento varia de cluster para cluster, por exemplo  nginx`